Add Markdown Links to README

bnb opened the pull request:
Added Markdown links to the README.md file for parts of the repo.

TierneyC committed:
Add Markdown Links to README

Added Markdown links to the README.md file for parts of the repo.

Fishrock123 commented on README.md:
```diff
@@ -4,19 +4,19 @@
-- [./wg-meetings] is an archive of the meeting minutes from this project's
-  Working Group (see [./GOVERNANCE.md]()).
+- [./wg-meetings](https://github.com/iojs/website/tree/master/wg-meetings) is an archive of the meeting minutes from this project's
+  Working Group (see [./GOVERNANCE.md](https://github.com/iojs/website/blob/master/GOVERNANCE.md)).
```
master --> v1.x

Fishrock123 replied:
EDIT: oops, thought this was to core's repo

Fishrock123 replied:
Shouldn't these be local links, so that they work properly in a dev environment?

TierneyC committed:
Testing local paths on GitHub

Trying out local paths to see if they work from the README on GitHub.

TierneyC committed:
Fixing local paths to absolute

Local paths get destroyed on GitHub. That's what led to there being no
links in the first place--`[./gulp](./gulp)` ends up as `[./gulp]()`.

bnb commented:
@Fishrock123 No, as they get destroyed through git/GitHub. One or the other (probably GitHub) removes `(./file.md)` from the Markdown link, leaving `()`. That''s what led to how the repository currently is.

Fishrock123 commented:
I know github allows local links. I bet @therebelrobot just forgot them. :)

TierneyC committed:
Fix YouTube Link

The old YouTube link went to a Livestream page. The new link goes to the
correct recording.

bnb commented:
Hmm. It appears I don't know how to use Git, and included another fix I did, intended to be separate, in this pull request. Is there a way I can I remove that, since this will probably be closed without merge?

TierneyC committed:
Revert "Fixing local paths to absolute"

This reverts commit 5b16e1c3f7e08714097e8f23a60c99db8ff43603.

bnb commented:
@Fishrock123 Okay! I just wanted to make sure they get linked properly. :)

TierneyC committed:
Revert "Testing local paths on GitHub"

This reverts commit 631e7da75951e70707630529fa6a687c1553b766.
